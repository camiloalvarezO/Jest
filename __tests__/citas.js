import Citas from "../js/classes/Citas";
// import jsdom from "jsdom";

// global.document = jsdom.JSDOM("<!doctype html><html><body></body></html>");
// global.window = global.document.defaultView;


describe('prueba a la clase de citas',()=>{

    const citas = new Citas()
    test('creación y validación de citas',()=>{
        const citaObj = {
            mascota: 'gatito',
            propietario: 'camilo',
            telefono: '30214',
            fecha: '24-06-2024',
            hora:'05:00',
            sintomas: 'dormido'
        }

        citaObj.id = Date.now();

        citas.agregarCita(citaObj);

        expect(citas).toMatchSnapshot()
    })
})