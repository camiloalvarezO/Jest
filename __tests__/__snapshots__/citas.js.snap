// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prueba a la clase de citas creación y validación de citas 1`] = `
Citas {
  "citas": [
    {
      "fecha": "24-06-2024",
      "hora": "05:00",
      "id": 1719464283133,
      "mascota": "gatito",
      "propietario": "camilo",
      "sintomas": "dormido",
      "telefono": "30214",
    },
  ],
}
`;
